web/src/app.css

```
@@ -115,6 +115,10 @@
 		@apply border-border outline-ring/50;
 	}
 	body {
-		@apply bg-background text-foreground;
+		@apply bg-background text-foreground min-h-screen antialiased;
+		background-image:
+			radial-gradient(circle at top, rgb(13 186 249 / 0.14), transparent 32%),
+			radial-gradient(circle at 20% 20%, rgb(20 184 166 / 0.12), transparent 24%);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)